### ISSUE TITLE
Better solution to breaking up speech to phones

### ISSUE THREAD

**Issue by kolodny** [kolodny/torah#3]
Ideally if we can get this to work locally in the browser that would be best. This would probably require training Tensorflowjs on mostly English speech since that's really the most common user of the app. We can use another program to convert the labeled transcript to phones and use that as our labels. We'll also need to augment the training data with some davening or something that contains english speaking people pronouncing the ches (and maybe tzaddik) sounds.
